Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -640,6 +655,10 @@ public class MapViewModel : BaseViewModel {
         /// Action to execute when the device location has updated
         /// </summary>
         private async void MyLocationUpdated(Location newLocation) {
+            bool wasWithinBoundary = mapModel.PlayingUser.Location == null
+                || mapModel.GameBoundary.Contains(mapModel.PlayingUser.Location);
+            bool isWithinBoundary = mapModel.GameBoundary.Contains(newLocation);
```

Comment:
```suggestion
            ShowOutsideBoundaryScreen = mapModel.GameBoundary.Contains(newLocation);
```
Waarom niet gewoon dit? Dan plaats je deze regel code net voor de if-statment. Dan hoeft er alleen gecontroleerd te worden of er meldingen moeten worden gestuurd.

Reply:
Ik heb het aangepast, maar de isWithinBounds variabel nog wel gehouden omdat dit het beter leesbaar maakt naar mijn mening.